script.js

```diff
@@ -7,11 +7,11 @@ window.onload = dom;
 let addTodo = () => {
   const todoInput = getTodoInput();
   const todoText = todoInput.value;
-  const newTodo = new NewTodo(todoText);
-  newTodo.addToModel(todoText);
+  const newItem = new TodoItem(todoText);
```

Comment:
please rename the `new` part of the name since you dont want to have

`const newFoo = new NewFoo()`